recipes/crashpad/all/conanfile.py

```diff
@@ -0,0 +1,260 @@
+
+        tools.replace_in_file(os.path.join(self._source_subfolder, "third_party", "mini_chromium", "mini_chromium", "build", "win_helper.py"),
+                              "print line", "print(line)")
+        tools.replace_in_file(os.path.join(self._source_subfolder, "third_party", "mini_chromium", "mini_chromium", "build", "win_helper.py"),
+                              "print result", "print(result)")
+
+        if self.settings.compiler == "gcc":
+            # Remove gcc-incompatible compiler arguments
+            for comp_arg in ("-Werror", "-Wheader-hygiene", "-Wnewline-eof", "-Wstring-conversion", "-Wexit-time-destructors", "-fobjc-call-cxx-cdtors", "-Wextra-semi", "-Wimplicit-fallthrough"):
+                tools.replace_in_file(toolchain_path,
+                                      "\"{}\"".format(comp_arg), "\"\"")
+
+        autotools = AutoToolsBuildEnvironment(self)
+        extra_cflags = autotools.flags + ["-D{}".format(d) for d in autotools.defines]
+        extra_cflags_c = []
+        extra_cflags_cc = autotools.cxx_flags
+        extra_ldflags = autotools.link_flags
+        if self.options.get_safe("fPIC"):
+            extra_cflags.append("-fPIC")
+        extra_cflags.extend("-I'{}'".format(inc) for inc in autotools.include_paths)
+        extra_ldflags.extend("-L'{}'".format(libdir) for libdir in autotools.library_paths)
+        if self.settings.compiler == "clang":
+            if self.settings.compiler.get_safe("libcxx"):
+                stdlib = {
+                    "libstdc++11": "libstdc++",
+                }.get(str(self.settings.compiler.libcxx), str(self.settings.compiler.libcxx))
+                extra_cflags_cc.append("-stdlib={}".format(stdlib))
+                extra_ldflags.append("-stdlib={}".format(stdlib))
+        gn_args = [
+            "host_os=\\\"{}\\\"".format(self._gn_os),
+            "host_cpu=\\\"{}\\\"".format(self._gn_arch),
+            "is_debug={}".format(str(self.settings.build_type == "Debug").lower()),
+            "crashpad_http_transport_impl=\\\"{}\\\"".format(self._http_transport_impl),
+            "crashpad_use_boringssl_for_http_transport_socket={}".format(str(self.options.get_safe("with_tls", False) != False).lower()),
+            "extra_cflags=\\\"{}\\\"".format(" ".join(extra_cflags)),
+            "extra_cflags_c=\\\"{}\\\"".format(" ".join(extra_cflags_c)),
+            "extra_cflags_cc=\\\"{}\\\"".format(" ".join(extra_cflags_cc)),
+            "extra_ldflags=\\\"{}\\\"".format(" ".join(extra_ldflags)),
+        ]
+        with tools.chdir(self._source_subfolder):
+            with self._build_context():
+                self.run("gn gen out/Default --args=\"{}\"".format(" ".join(gn_args)), run_environment=True)
+                for target in ("client", "minidump", "crashpad_handler", "snapshot"):
+                    # FIXME: Remove verbose once everything is working hunky dory
+                    self.run("ninja -v -C out/Default {target} -j{parallel}".format(
```

Comment:
SHould we keep -v now that the recipe is mature?

Reply:
*mature* is a big word.
But sure, let's drop it.

Reply:
We can keep it until this is ready to get merged haha
Tbh I don't really mind, but since there is a hook that catches this for CMake and wont let you merge with verbose builds I guess we should keep this recipe in line with that policy.